src/coreModules/api/specEndpointFactory/index.js

```diff
@@ -0,0 +1,34 @@
+import openApi from 'dina-schema/build/openApi.json'
+
+const buildOperationIdPathnameMap = openApiSpec => {
+  const map = {}
+  Object.keys(openApiSpec.paths).forEach(pathname => {
+    const verbs = openApiSpec.paths[pathname]
+    Object.keys(verbs).forEach(verbName => {
+      const verb = verbs[verbName]
+      if (verb.operationId)
+        map[verb.operationId] = {
+          pathname,
+          verbName,
+        }
+    })
+  })
+  return map
+}
+
+const map = buildOperationIdPathnameMap(openApi)
```

Comment:
would suggest to pass `openApiSpec.paths` here. 

or even `import { paths } from '...openApi.json'` (and maybe call the parameter to `buildOperationIdPathnameMap` `pathnames` instead